owners/app.yaml

```diff
@@ -13,3 +13,6 @@
 
 runtime: nodejs
 env: flex
+
+automatic_scaling:
+  max_num_instances: 1
```

Comment:
Also `min_num_instances: 1`? 🙃

Reply:
I don't think that's needed based on examples I saw in docs, but added anyway to be sure :+1: